extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -714,3 +733,13 @@ export class AmpSidebar extends AMP.BaseElement {
 AMP.extension('amp-sidebar', '0.1', (AMP) => {
   AMP.registerElement('amp-sidebar', AmpSidebar, CSS);
 });
+
+/**
+ * @param {!Element} element
+ * @return {boolean}
+ */
+function excludeFromSwipeClose(element) {
+  return (
+    element.nodeName === 'INPUT' && element.getAttribute('type') === 'range'
```

Comment:
`element.nodeName.toLowerCase() === 'input'` (as per this old article https://johnresig.com/blog/nodename-case-sensitivity/ there are some corner cases, better to guard against them)

Reply:
Thanks!  Updated in the latest revision.  Thanks for the review!